src/crates.rs

```diff
@@ -112,14 +114,24 @@ pub fn sync_crates_files(
         repo.diff_tree_to_tree(Some(&master_tree), Some(&origin_master_tree), None)?
     };
 
-    // Run one pass to figure out a total count
-    let mut count = 0;
+    let mut changed_crates = Vec::new();
+    let mut removed_crates = Vec::new();
+
+    let pb = ProgressBar::new_spinner()
+        .with_style(
+            ProgressStyle::default_bar()
```

Comment:
So it appears that Indicatif changed (or has always had) the defaults on their progress bar set to automatically clear when finished: https://github.com/mitsuhiko/indicatif/blob/main/src/style.rs#L463

So that's a bit of a pain. In that case, would you be able to change the `ProgressStyle`s and add `.on_finish(ProgressFinish::AndLeave)` on this progress bar and the other progress bars in the codebase?